pkg/asset/installconfig/powervs/validation_test.go

```diff
@@ -113,6 +113,18 @@ var (
 			ID:   &validRG,
 		},
 	}
+	regionWithPER    = "dal10"
+	regionWithoutPER = "foo99"
+	mapWithPERFalse  = map[string]bool{
+		"disaster-recover-site": true,
+		"power-edge-router":     false,
+		"vpn-connections":       true,
+	}
+	mapWithPERTrue = map[string]bool{
+		"disaster-recover-site": true,
+		"power-edge-router":     true,
+		"vpn-connections":       true,
+	}
```

Comment:
For completeness, a test case where "power-edge-router" is not available.

Reply:
Addressed in the latest force-push. PTAL.